skills_repository/alfred_skill.py

```diff
@@ -1,5 +1,5 @@
 from abc import ABC, abstractmethod, abstractproperty
-
+IGNORE = True
```

Comment:
There may be other modules that skills depend on, so we should rely on something other than a definition in each module probably.